Core/include/Acts/Utilities/AnnealingUtility.hpp

```diff
@@ -30,8 +30,7 @@ class AnnealingUtility {
   /// @brief The configuration struct
   struct Config {
     // Config constructor with default temperature list: {64.,16.,4.,2.,1.5,1.}
```

Comment:
Should we update the comment too?
```suggestion
    // Config constructor with default temperature list: {1.}
```